views/404.ejs

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title><%-data.name%></title>
+    <style>
+    * {
+      margin: 0;
+      padding: 0;
+      outline: 0;
+    }
+
+    body {
+      padding: 80px 100px;
+      font: 13px "Helvetica Neue", "Lucida Grande", "Arial";
+      background: #ECE9E9 -webkit-gradient(linear, 0% 0%, 0% 100%, from(#fff), to(#ECE9E9));
+      background: #ECE9E9 -moz-linear-gradient(top, #fff, #ECE9E9);
+      background-repeat: no-repeat;
+      color: #555;
+      -webkit-font-smoothing: antialiased;
+    }
+    h1, h2 {
+      font-size: 22px;
+      color: #343434;
+    }
+    h1 em, h2 em {
+      padding: 0 5px;
+      font-weight: normal;
+    }
+    h1 {
+      font-size: 60px;
+    }
+    h2 {
+      margin: 10px 0;
+    }
+    ul li {
+      list-style: none;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="wrapper">
+      <h1><%-data.name%></h1>
+      <h2><em><%-data.statusCode%></em> <%-data.message%></h2>
+      <%//for debug mode to display appended data
+        for (var prop in data ) {
+          if (['name', 'statusCode', 'status', 'message', 'stack'].indexOf(prop) == -1) {
```

Comment:
here I am trying to have the ones displayed with standard format not go through the loop, so if title and message is used I don't loop through it again